# WildHostEcoliMexico


## Metadata and intermediate for "Genomic epidemiology and evolution of Escherichia coli in wild animals in Mexico"
### Authors: Robert Murphy, Martin Palm, Ville Mustonen, Jonas Warringer, Anne Farewel, Leopold Parts,Danesh Moradigaravand


## Abstract 
Escherichia coli is a common bacterial species in the gastrointestinal tracts of warm-blooded animals and humans. Pathogenic and antimicrobial resistance in E. coli may emerge via host switching from animal reservoirs. Despite its potential clinical importance, knowledge of the population structure of commensal E. coli within wild hosts and the epidemiological links between E. coli in non-human hosts and E. coli in humans is still scarce. In this study, we analysed the whole genome sequencing data of a collection of 119 commensal E. coli recovered from the guts of 55 mammal and bird species in Mexico and Venezuela in the 1990s. We observed low concordance between the population structures of E. coli colonizing wild animals and the phylogeny, taxonomy and ecological and physiological attributes of the host species, with distantly related E. coli often colonizing the same or similar host species and distantly related host species often hosting closely related E. coli. We found no evidence for recent transmission of E. coli genomes from wild animals to either domesticated animals or humans. However, multiple livestock- and human-related virulence factor genes were present in E. coli of wild animals, including virulence factors characteristic for Shiga toxin-producing E. coli (STEC) and atypical enteropathogenic E. coli (aEPEC), where several isolates from wild hosts harboured the locus of enterocyte effacement (LEE) pathogenicity island. Moreover, E. coli in wild animal hosts often harboured known antibiotic resistance determinants, including against ciprofloxacin, aminoglycosides, tetracyclines and beta-lactams, with some determinants present in multiple, distantly related E. coli lineages colonizing very different host animals. We conclude that although the genome pools of E. coli colonizing wild animal and human gut are well separated, they share virulence and antibiotic resistance genes and E. coli underscoring that wild animals could serve as reservoirs for E. coli pathogenicity in human and livestock infections.  


## Description of files:
### SupplementalTableS1.csv: 
Samples specification, serotypes, associated pathovars and virulence and AMR genes
### SupplementalTableS2.csv: 
Samples specification and associated metadata for external isolates from Enterobase, shown in Supplemental Figure S1.
### SupplementalTableS3.csv: 
Table of Ka/Ks values and functional annotations for genes present in >70% of isolates  
### AlignmentBayesianAnalysis.aln: 
SNP alignemnt for strains used in the Bayesian anslysis
### core_genome_snps_alignment.aln: 
SNP alignemnt for variants in the core genome of 119 strains of wild hosts origins
### gene_presence_absence.csv: 
Presence absence gene for the pan-genome of 119 strains file from Roary 
